bih-cluster/docs/storage/storage-migration.md

```diff
@@ -0,0 +1,175 @@
+## What is going to happen?
+Files on the cluster's main storage `/data/gpfs-1` aka. `/fast` will move to a new filesystem.
+That includes users' home directories, work directories, and workgroup directories.
+Once files have been moved to their new locations, `/fast` will be retired.
+
+## Why is this happening?
+`/fast` is based on a high performance proprietary hardware (DDN) & filesystem (GPFS).
+The company selling it has terminated support which also means buying replacement parts will become increasingly difficult.
+
+## The new storage
+There are *two* filesystems set up to replace `/fast`, named *Tier 1* and *Tier 2* after their difference in I/O speed:
+
+- **Tier 1** is faster than `/fast` ever was, but it only has about 75  % of its usable capacity.
+- **Tier 2** is not as fast, but much larger, almost 3 times the current usable capacity.
+
+The **Hot storage** Tier 1 is reserved for large files, requiring frequent random access.
+Tier 2 (**Warm storage**) should be used for everything else.
+Both filesystems are based on the open-source, software-defined [Ceph](https://ceph.io/en/) storage platform and differ in the type of drives used.
+Tier 1 or Cephfs-1 uses NVME SSDs and is optimized for performance, Tier 2 or Cephfs-2 used traditional hard drives and is optimised for cost.
+
+So these are the three terminologies in use right now:
+- Cephfs-1 = Tier 1 = Hot storage
+- Cephfs-2 = Tier 2 = Warm storage
+
+### Snapshots and Mirroring
+Snapshots are incremental copies of the state of the data at a particular point in time. 
+They provide safety against various "Oups, did I just delete that?" scenarios, meaning they can be used to recover lost or damaged files.
+
+Depending on the location and Tier, Cephfs utilises snapshots in differ differently.
+Some parts of Tier 1 and Tier 2 snapshots are also mirrored into a separate fire compartment within the datacenter to provide an additional layer of security.
+
+| Tier | Location                 | Path                         | Retention policy                | Mirrored |
+|:-----|:-------------------------|:-----------------------------|:--------------------------------|---------:|
+|    1 | User homes               | `/data/cephfs-1/home/users/` | Hourly for 48 h, daily for 14 d | yes      |
```

Comment:
Do we need to mention it is backed-up (assuming it is, of course)?

Reply:
Are there actually backups besides the mirrored snapshots? I'm not sure, we have to clarify.